docs/preview/03-Guidance/use-with-dotnet-and-functions.md

```diff
@@ -12,8 +12,11 @@ We encourage you to follow the standard [Serilog instructions](https://github.co
 Some aspects we would like to highlight are:
 
 - Make sure to call [`UseSerilog`](https://www.nuget.org/packages/Serilog.AspNetCore) when creating a `IHostBuilder`
+- Remove the default Microsoft's `ApplicationInsightsLoggerProvider` via the `RemoveMicrosoftApplicationInsightsLoggerProvider` extension
 - Remove default for logging including its configuration in `appsettings.json` *(if applicable)*
```

Comment:
What exactly is meant with this last bullet ? (remove default for logging ... ) ?

Reply:
Was also thinking about that. I think it's the `"applicationInsights"` parts in the `appsettings.json`, but I can't quite remember why it was here.

Reply:
Should we remove this bullet, as it is only causing confusion ?
Or should we try to clarify it a bit.   The `Logging` [section ](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0#configure-logging)in appsettings.json is indeed not used when we use Serilog, as it uses another section.

Maybe we should modify this piece to make it more clear.  Something like:
```suggestion
- Remove the `Logging` section from the `appsettings.json` *(if applicable)* as this is not used by Serilog
```

Reply:
Yeah, ok! It's the `Serilog` section in the `appsettings.json` file that Serilog can use with the `ReadFrom.Configuration(...)`, right?

Reply:
Yes indeed.